googleapis/api/experimental.pb.go

```diff
@@ -0,0 +1,57 @@
+// Code generated by protoc-gen-go.
+// source: google/api/experimental/experimental.proto
+// DO NOT EDIT!
+
+package api
```

Comment:
Hi Michael,

Is this issue blocking your PR ? If so, I can look into this tomorrow, else may be before end of this month.